sdk/keyvault/azure-keyvault-secrets/src/main/java/com/azure/security/keyvault/secrets/SecretClient.java

```diff
@@ -473,9 +475,13 @@ public Secret restoreSecret(byte[] backup) {
      * call {@link SecretClient#getSecret(SecretBase baseSecret)} . This will return the {@link Secret secret} with value included of its latest version.</p>
      * {@codesnippet com.azure.security.keyvault.secretclient.listSecrets}
      *
-     * @return A {@link List} containing {@link SecretBase} of all the secrets in the vault. The {@link SecretBase} contains all the information about the secret, except its value.
+     * <p>It is possible to get full secrets with values from this information. Loop over all the {@link SecretBase secret} by page and
```

Comment:
Description is fine, But the header should stay too.
`<p><strong>Code Samples to iterate over secrets by page</strong></p>`